pipeline_steps/common.groovy

```diff
@@ -30,13 +30,6 @@ def void create_workspace_venv(){
     # UG-613 change TMPDIR to directory with more space
     export TMPDIR="/var/lib/jenkins/tmp"
 
-    # UG-612 Install pip, setuptools, and wheel on the host
```

Comment:
Removed both the pip install lines from the last task in setup_jenkins_slave, as thats covered by create_workspace_venv, which is called before setup_jenkins_slave is run.